### DISCUSSION TITLE
BEP-0004 "Changelogs"

### DISCUSSION THREAD

**Discussion by michaelweinold** [brightway-lca/enhancement-proposals#26] [BEP Discussion]
[BEP004](https://github.com/brightway-lca/enhancement-proposals/blob/main/proposals/0004_changelogs.md)

**Comment by tngTUDOR** [comment]
It's unclear from the proposal if we are to suggest `rst` as the file format for the change logs.

My personal view is that changelogs are fine if they are simple markdown files. However, integration with the broader documentation projects of brightway might be a bit better if we use `rst`.

@michaelweinold if you are proposing to use `rst` specifically, this must be explicit in the BEP.

**Reply by michaelweinold** [reply]
Agreed, thank you for pointing this out. The relevant update to the BEP-004 PR (https://github.com/brightway-lca/enhancement-proposals/pull/27) is pending resolution of https://github.com/brightway-lca/brightway-documentation/issues/18 (introduction of Markdown alongside reStructuredText, where appropriate).

**Reply by michaelweinold** [reply]
BEP-004 further needs more explicit specifications related to the syntax of changelog entries. https://keepachangelog.com/en/1.0.0/ is not very explicit either, but does provide a template.

**Comment by michaelweinold** [comment]
...adopt https://semver.org/?

**Comment by tngTUDOR** [comment]
@michaelweinold the BEP advocates in favor of adopting a single filename to document changes: `CHANGES.md` . However, the keepachangelog best practice explicitly calls for calling the file "CHANGELOG.md". 

visualization of markdown files in github works just fine, and the cookiecutter-brightwaylib already uses "CHANGES.md" as  the standard file, so I vote in favor of keeping "CHANGES.md", considering that in my personal opinion _CHANGES_ talks more to a user, while _CHANGELOG_ would talk more to a developper, and this BEP is inscribed in the first perspective.
